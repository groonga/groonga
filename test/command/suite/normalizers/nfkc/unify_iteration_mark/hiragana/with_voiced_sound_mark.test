#@add-substitution /"version",\s"(?:.+?)"/ "\"version\", \"#{ENV['NFKC'] || '10.0.0'}\"" "\"version\", \"NFKC_VERSION\""
normalize \
  'NormalizerNFKC("unify_iteration_mark", true)' \
  "こゝで、一つづゝ分ける。" \
  WITH_TYPES
#@remove-substitution /"version",\s"(?:.+?)"/
