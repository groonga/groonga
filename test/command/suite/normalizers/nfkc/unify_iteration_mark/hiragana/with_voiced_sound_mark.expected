normalize   'NormalizerNFKC("unify_iteration_mark", true,                   "version", "NFKC_VERSION")'   "こゝで、一つづゝ分ける。"   WITH_TYPES
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "ここで、一つづつ分ける。",
    "types": [
      "hiragana",
      "hiragana",
      "hiragana",
      "symbol",
      "kanji",
      "hiragana",
      "hiragana",
      "hiragana",
      "kanji",
      "hiragana",
      "hiragana",
      "symbol",
      "null"
    ],
    "checks": []
  }
]
